js/algoliasearch/instantsearch.js

```diff
@@ -59,6 +59,22 @@ document.addEventListener("DOMContentLoaded", function (event) {
 			}
 		};
 		
+		if (algoliaConfig.request.path.length > 0 && window.location.hash.indexOf('categories.level0') === -1) {
+			if (algoliaConfig.areCategoriesInFacets === false) {
```

Comment:
Makes sense @Inwerpsel,
thanks for the review. I changed it and will merge it when tests passes ...